CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+We welcome contributions to Valhalla. If you would like to report an issue, or even better fix an existing one, please use the [Valhalla issue tracker](https://github.com/valhalla/valhalla/issues) on GitHub.
+
+There are many ways to make meaningful contributions to the project:
+- file issues & bugs with **clear and easily reproducible instructions**
+- contribute bug fixes
+- contribute feature implementations, for your own sake **only after opening an issue asking for feedback**
+- improve translations on [Transifex](https://www.transifex.com/valhalla/valhalla-phrases/locales-en-us-json--transifex/)
+
+## Community Values
+
+We ask that you are respectful when contributing to Valhalla or engaging with our community. As a community, we appreciate the fact that contributors might be approaching the project from a different perspective and background. We hope that beginners as well as advanced users will be able to use and contribute back to Valhalla. We want to encourage contributions and feedback from all over the world, which means that English might not be a contributor's native language, and sometimes we may encounter cultural differences. Contructive disagreements can be essential to moving a project forward, but disrespectful language or behavior will not be tolerated.
+
+Above all, be patient, be respectful, and be kind!
```

Comment:
Took that from Mapzen Pelias days, I liked the text.

Reply:
my vote is to leave it out. no offense to pelias of course but i can't see what information this adds or what purpose it serves.

let me explain. it basically just says "don't be a jerk or else". but even that abbreviated version is fairly useless. i can see a few lines of reasoning. either you are not a jerk in which case the text above does not apply to you OR you are a jerk. if you are a jerk are you really going read the readme and say ah damn they told me not to be a jerk, i guess ill find another project? i highly doubt it.

i guess there could be people who are undecided on their jerkhood but i think the only category of people who fall into this (people who need to be reminded not to be jerks) are children. i doubt we need to worry about children coming to the project that are also young enough that they need to be told to act nice in which case we are, at best, condencending to the adults that come here.

the only other purpose i can see it serving is as a pat on the back to ourselves that we are not jerks which i think is better shown through our own actions rather than asserted in our readme.

Reply:
hahahaha totally fine with me, I'll remove it.